lib/puppet/pops/lookup/lookup_adapter.rb

```diff
@@ -378,8 +378,8 @@ def initialize_module_provider(lookup_invocation, module_name)
       mp
     else
       unless Puppet[:strict] == :off
-        msg = "Defining \"data_provider\": \"#{provider_name}\" in metadata.json is deprecated"
-        msg += ". A '#{HieraConfig::CONFIG_FILE_NAME}' file should be used instead" if mp.nil?
+        msg = _("Defining \"data_provider\": \"%{name}\" in metadata.json is deprecated") % { name: provider_name }
```

Comment:
@Magisus metadata.json refers to a filename, should it be left out of translation?